Packs/SendGrid/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+##### SendGrid
+Added 6 new commands
+- sg-create-list
+- sg-delete-list
+- sg-get-all-lists
+- sg-get-list-by-id
+- sg-get-list-contact-count-by-id
+- sg-update-list-name
+- Updated the Docker image to: *demisto/sendgrid:1.0.0.23423*.
```

Comment:
looks like you updated the docker image to "demisto/sendgrid:1.0.0.25330".

Reply:
Yes, it got updated to latest version. Can you please help to push this change?